src/logger/loggerApi.c

```diff
@@ -1435,8 +1435,16 @@ int api_addTrackDb(Serial *serial, const jsmntok_t *json){
 		Track track;
 		const jsmntok_t *trackNode = findNode(json, "track");
 		if (trackNode != NULL) setTrack(trackNode + 1, &track);
-		add_track(&track, index, mode);
-		return API_SUCCESS;
+		int result = add_track(&track, index, mode);
+		if (result == TRACK_ADD_RESULT_OK){
```

Comment:
Looks like spaces were used when spaces where tabs were supposed to be there.  If/else is on different line